src/chrome/content/rules/tfaw.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="tfaw.com">
+	<target host="www.tfaw.com"/>

```

Comment:
Add `<target host="tfaw.com" />`.

Reply:
I commented it out because it was causing travis to fail and the site seems to always use www instead. I can add it back if you'd prefer to have it in and let the test fail.